api-doc/rest_api.md

```diff
@@ -385,4 +384,28 @@ Return the list of find properties supported by the application.  These properti
     "iri": "https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/explanatoryNote"
   }
 ]
+```
+
+## 11. Missing Imports
```

Comment:
@patrycjamia Please take a look how the description is prepared for example for `8. Actuator endpoints` and use the same format. I know that old descriptions aren't updated yet but we should use the new format from now on.